src/bandit-evaluator.ts

```diff
@@ -23,6 +23,45 @@ export class BanditEvaluator {
   private readonly assignmentShards = BANDIT_ASSIGNMENT_SHARDS; // We just hard code this for now
   private readonly sharder: Sharder = new MD5Sharder();
 
+  public evaluateBestBandit(
```

Comment:
Minor but `evaluateBestBandit` makes it seem like we're picking the best of many bandits--what about `evaluateBanditBestAction()`?